debian10/Dockerfile

```diff
@@ -2,29 +2,30 @@ ARG BUILD_FROM=debian:10
 FROM ${BUILD_FROM}
 
 RUN apt-get update && apt-get install -y \
-    procps \
-    git \
-    wget \
-    zip \
-    unzip \
-    vim \
+    autoconf \
+    automake \
+    autotools-dev \
+    bison \
     build-essential \
-    debhelper \
     curl \
-    libssl-dev \
-    automake \
-    autoconf \
-    libncurses5-dev \
-    gcc \
+    debhelper \
+    flex \
     g++ \
-    make \
-    zlib1g-dev \
+    gcc \
+    git \
+    jq \
     libffi-dev \
+    libncurses5-dev \
+    libssl-dev \
+    make \
+    procps \
     unixodbc \
     unixodbc-dev \
-    jq \
-    flex \
-    bison
+    unzip \
+    vim \
```

Comment:
?

Reply:
Are you wondering why vim is there? I do not know.

Reply:
Yes, seems strange. But it's legacy code, so no questions to the PR